docs/Catalogs.adoc

```diff
@@ -0,0 +1,279 @@
+= Using Operator Catalog Mirrors
+
+== Overview
+
+Operators in OpenShift change quite frequently - even within operator subscription channels. Also the available ClusterServiceVersions for any given operator change very frequently with older versions being removed from the catalog - even though the container images making up the operator and/or managed applications still exist in the various container registries.
+
+In order to have a predictable environment for demos or classroom environments it is preferable to install exactly the operator version that the demo / lab was written for.
+
+One way to achieve this is by creating an Operator Catalog mirror. There are 4 catalogs in OpenShift (as of July 2020):
+
+* Red Hat Operators
+* Certified Operators
+* Community Operators
+* Marketplace Operators
+
+Mirroring a catalog means creating a snapshop of a catalog at a state in time. This snapshot is created as a container image and needs to be hosted in a (public) container registry.
+
+While the online catalog may be updated with newer releases of any given operator the mirrored catalog will always contain the same cluster service versions for any given operator.
+
+The process for mirroring an operator catalog is:
```

Comment:
This section is confusing. You are covering everything from creating a repo to deploying the operator. If you are explaining the process for mirroring an operator catalog, keep it to just that.

Reply:
Reworded